packages/cli/package.json

```diff
@@ -57,5 +57,8 @@
         "webpack": "^5.28.0",
         "webpack-cli": "^4.6.0",
         "xml-js": "^1.6.11"
+    },
+    "dependencies": {
+        "chalk": "^4.1.0"
```

Comment:
move it to `devDependencies`, everything is webpacked into one file in the cli, so no need for npm to install this as a dependency

Reply:
^just this comment, then its okay to merge 😄

Reply:
hmmm ... that's another point which I would challenge a bit ... webpak is cool but very unintuitive. Why devDependency if it is a real dependency ? How to debug it if needed ? and etc ... honestly I'd suggest adjusting the devflow, bow it is not transparent. However this is a separate subject, maybe for slack :)